docs/Installation.md

```diff
@@ -65,6 +65,10 @@ sudo cpan -i Zonemaster::Backend
 Check the [declaration of prerequisites] to make sure your preferred combination
 of operating system version and database engine version is supported.
 
+# Add Zonemaster user

```

Comment:
Please remove the `#` level 1 heading marker and append a colon instead.

Reply:
OK